profiler/__openerp__.py

```diff
@@ -0,0 +1,51 @@
+# coding: utf-8
+# ============================================================================
+#                                                                             =
+#    profiler module for OpenERP, cProfile integration for Odoo/OpenERP
+#    Copyright (C) 2014 Anybox <http://anybox.fr>
+#                                                                             =
+#    This file is a part of profiler
+#                                                                             =
+#    profiler is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License v3 or later
+#    as published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#                                                                             =
+#    profiler is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License v3 or later for more details.
+#                                                                             =
+#    You should have received a copy of the GNU Affero General Public License
+#    v3 or later along with this program.
+#    If not, see <http://www.gnu.org/licenses/>.
+#                                                                             =
+# =============================================================================
+{
+    'name': 'profiler',
+    'version': '8.0.1.0.0',
+    'category': 'devtools',
+    'author': 'Odoo Community Association (OCA),Georges Racinet,Vauxoo',
+    'website': 'https://odoo-community.org,'
+    'http://anybox.fr,https://www.vauxoo.com/',
```

Comment:
Hi guys, first, I'd like to tell that I'm glad that some code I initiated should find its way into OCA !

As far as the website goes, I'm fine with keeping only https://odoo-community.org, it's the rule of the game we're (well mostly you) are playing.